.env.example

```diff
@@ -59,6 +59,9 @@ ELASTICSEARCH_VERIFY_CERTS=FALSE
 REDIS_HOST=<REDIS HOSTNAME>
 REDIS_PORT=<REDIS PORT>
 
+# Celery
+CELERY_CLEAR_EXPIRED_SESSION_IDS=<CELERY_CLEAR_EXPIRED_SESSION_IDS>
```

Comment:
Can we change this variable name to be something like `CELERY_CLEAR_EXPIRED_SESSION_IDS_INTERVAL` that way we know exactly what it refers to?